yt_dlp/extractor/twitch.py

```diff
@@ -455,20 +463,22 @@ def _extract_info_gql(self, info, item_id):
             vod_id = 'v%s' % vod_id
         thumbnail = url_or_none(info.get('previewThumbnailURL'))
         is_live = None
+        thumbnails = None
```

Comment:
we can revert most of the changes in this class and then inline the new method in the return
```suggestion
```